db/migrate/migrate.go

```diff
@@ -76,6 +141,10 @@ func Migrate(db *gorm.DB) error {
 						return err
 					}
 
+					if err := tx.Migrator().DropTable("permissions"); err != nil {
```

Comment:
```suggestion
					if err := tx.Migrator().DropTable("permissions"); err != nil {
```
This is a bit bugged due to git. But the rollback for the admin users migration should only delete the admin users from the users table.